MoltenVK/MoltenVK/GPUObjects/MVKSwapchain.mm

```diff
@@ -426,28 +426,37 @@ static inline CIE1931XY VkXYColorEXTToCIE1931XY(VkXYColorEXT xy) {
 VkResult MVKSwapchain::getPastPresentationTiming(uint32_t *pCount, VkPastPresentationTimingGOOGLE *pPresentationTimings) {
 	if (_device->getConfigurationResult() != VK_SUCCESS) { return _device->getConfigurationResult(); }
 
+	VkResult res = VK_SUCCESS;
+
+	// Not allowed, but handle it with some valid error code to avoid crash.
+	if (pCount == nullptr) { return VK_ERROR_OUT_OF_HOST_MEMORY; }
+
```

Comment:
I have a slight preference for not doing error checking if the Vulkan spec says some behavior `must:` or `must: not` happen.

Reply:
Thanks for the review. I'll drop that check and update the commit message accordingly.

Reply:
Done.